DEV.md

```diff
@@ -0,0 +1,225 @@
+## Developer docs
+
+### Requirements
+
+Building Scala CLI requires JVM 17 to work properly.
+
+In theory, our build is able to download and install for its own needs JVM 17 on some OSes however it may not work in
+Intellij / Metals out of the box.
+
+The Scala CLI sources ship with Mill launchers, so that Mill itself doesn't need to be installed on your system.
+
+### Common commands
+
+#### Running the CLI from sources
+
+```bash
+./mill -i scala …arguments…
+```
+
+#### Run unit tests
+
+```bash
+./mill 'build-module.test' './mill 'build-module.test'
+```
+
+#### Run integration tests with the JVM launcher
+
+```bash
+./mill integration.test.jvm
+```
+
+Filter test suites with
+
+```bash
+./mill integration.test.jvm 'scala.cli.integration.RunTestsDefault.*'
+./mill integration.test.jvm 'scala.cli.integration.RunTestsDefault.Multiple scripts'
+```
+
+Pass the `--debug` option to debug integration tests:
```

Comment:
Describe more how developers can debug Scala CLI, when pass `--debug` option to integrate test.

Reply:
should be fixed